db/migrate/20200412232805_create_exception_hunter_error_groups.rb

```diff
@@ -0,0 +1,14 @@
+class CreateExceptionHunterErrorGroups < ActiveRecord::Migration[6.0]
+  def change
+    enable_extension :pg_trgm
+
+    create_table :exception_hunter_error_groups do |t|
+      t.string :error_class_name, null: false
+      t.string :message
```

Comment:
Shouldn't this be not null as well?

Reply:
Also let's re-refresh what's the difference between this message and `Error` message, shall we?